deploy.sh

```diff
@@ -1,4 +1,4 @@
-export NODE_ENV=production
+export NODE_ENV=development
```

Comment:
why did you change it? it is mainly used for server usage.

Reply:
actually, this one was unintentional. I tested it on localhost and forgot to set it back